tests/vmi_lifecycle_test.go

```diff
@@ -631,6 +634,94 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		})
 
+		Context("with default cpu model", func() {
+			var cfgMap *k8sv1.ConfigMap
+			var originalData map[string]string
+			var options metav1.GetOptions
+			var defaultCPUModelKey = "default-cpu-model"
+			var defaultCPUModel = "Conroe"
+			var vmiCPUModel = "SandyBridge"
+
+			//store old kubevirt-config
+			BeforeEach(func() {
+				cfgMap, err = virtClient.CoreV1().ConfigMaps(namespaceKubevirt).Get(kubevirtConfig, options)
+				Expect(err).ToNot(HaveOccurred())
+				originalData = cfgMap.Data
+			})
+
+			//replace new kubevirt-config with old config
+			AfterEach(func() {
+				cfgMap, err = virtClient.CoreV1().ConfigMaps(namespaceKubevirt).Get(kubevirtConfig, options)
+				Expect(err).ToNot(HaveOccurred())
+				cfgMap.Data = originalData
+				_, err = virtClient.CoreV1().ConfigMaps(namespaceKubevirt).Update(cfgMap)
+				Expect(err).ToNot(HaveOccurred())
+				_, err = syncKubevirtConfigMap()
```

Comment:
replace the sync function with a sleep. I explain the reasoning in another comment.